managed/services/dbaas/kubernetes/client/database/database.go

```diff
@@ -50,9 +50,13 @@ func NewForConfig(c *rest.Config) (*DatabaseClusterClient, error) {
 	config.UserAgent = rest.DefaultKubernetesUserAgent()
 
 	addToScheme.Do(func() {
-		dbaasv1.SchemeBuilder.AddToScheme(scheme.Scheme)
+		addToSchemeError=dbaasv1.SchemeBuilder.AddToScheme(scheme.Scheme)
 		metav1.AddToGroupVersion(scheme.Scheme, dbaasv1.GroupVersion)
 	})
+	if addToSchemeError != nil {
+		       return nil, addToSchemeError
+		   }
+
```

Comment:
Let's just mute linter here. Whole `dbass` package will be dropped soon.

Reply:
Hi @artemgavrilov  i made the changes please review